dpdd.tex

```diff
@@ -1264,7 +1265,7 @@ \subsection{Open Issues}
 
 \clearpage
 
-\section{Level 3 Data Products and Capabilities}
+\section{User Generated Data Products and Capabilities}
 \label{sec:level3}
 
 \emph{Level 3 capabilities} are envisioned to enable science cases that would greatly benefit from co-location of user processing and/or data within the LSST Archive Center. The high-level requirement for Level 3 is established in \S~3.5 of the LSST \SRD.
```

Comment:
Did you mean for these "Level 3" parts to be left in? I wonder if this entire section should be rejigged in terms of User Generated Data Products and LSST Science Platform.